docs/models.rst

```diff
@@ -79,3 +79,19 @@ It is also possible to have two concrete Heroku Connect models to Inherit from
 each other.
 
 .. _`Multi-table inheritance`: https://docs.djangoproject.com/en/stable/topics/db/models/#multi-table-inheritance
+
+
+Signals
+-------
+
+Standard `Django model signals`_ won't be triggered in case of changes coming from Salesforce. This is happening because
+Heroku Connect operates on a database level.
```

Comment:
You didn't add any line-breaks anywhere else, why here ;)
```suggestion
Standard `Django model signals`_ won't be triggered for changes made on Salesforce, because Heroku Connect operates on a database-level.
```